charts/iperf3/values.yaml

```diff
@@ -4,3 +4,6 @@ service:
   # externalIp:
   # the port to bind on the service, if enabled.
   port: 40000
+# uncomment and change this to set the number of replicas
+# default is 1 replica
+# replicas: 1
```

Comment:
please set the default in the values, not in the deployment via gotpl

Reply:
Done.